packages/webpack-module-federation/tests/defineConfig.test.ts

```diff
@@ -0,0 +1,455 @@
+/*
+build host:
+-
+
+
+build remote:
+
+
+*/
+
+import { defineBuildConfig as defineSwcBuildConfig, defineDevConfig as defineSwcDevConfig } from "@workleap/swc-configs";
```

Comment:
```suggestion
import { defineBuildConfig as defineSwcBuildConfig, defineDevConfig as defineSwcDevConfig } from "@workleap/swc-configs";
```

This is a strange comment